programmers/두-정수-사이의-합.js

```diff
@@ -0,0 +1,9 @@
+function solution(a, b) {
+  if (a > b) [a, b] = [b, a];
+  var answer = 0;
+  for (let i = a; i <= b; i++) {
+    answer += i;
+  }
```

Comment:
i는 보통 index로 많이 쓰입니다.
인덱스가 아닌 단순 숫자의 합이니 num이라는 변수명이 좋겠네요.
js에 range는 없는게 맞습니다. 적어도 제가 알기로는 :D
```suggestion
  for (let num = a; num <= b; num++) {
    total += num;
  }
```